ntbs-service/wwwroot/source/Components/ValidateInput.ts

```diff
@@ -21,11 +21,12 @@ const ValidateInput = Vue.extend({
             this.$emit("validate", event);
 
             const requestConfig = {
+                method: "post" as Method,
                 url: `${getValidationPath(this.$props.model)}Property`,
                 headers: getHeaders(),
-                params: {
+                data: {
                     "value": newValue,
-                    "shouldValidateFull": this.$props.shouldvalidatefull || false,
+                    "shouldValidateFull": this.$props.shouldvalidatefull.toLowerCase() == "true",
```

Comment:
The old implementation is weird here - I assume it was only working before because `shouldvalidatefull` could either be `"true"` or `null`?

Reply:
Yes basically